_config.yml

```diff
@@ -11,6 +11,7 @@ email: aframe@mozilla.com
 language:
 robots: all
 blog_title: Blog
+theme_color: black
```

Comment:
```<meta name="theme-color" content="black">```

https://html.spec.whatwg.org/multipage/semantics.html#meta-theme-color
https://developers.google.com/web/fundamentals/design-and-ui/browser-customization/theme-color